ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -36,6 +36,12 @@ class SegmentPillButton: UIButton {
             var configuration = UIButton.Configuration.plain()
             configuration.contentInsets = Constants.insets
             configuration.background.backgroundColor = .clear
+            let titleTransformer = UIConfigurationTextAttributesTransformer { incoming in
+                var outgoing = incoming
+                outgoing.font = UIFont.systemFont(ofSize: Constants.fontSize)
+                return outgoing
+            }
+            configuration.titleTextAttributesTransformer = titleTransformer
             self.configuration = configuration
         } else {
```

Comment:
should the else case have self.titleLabel?.font = UIFont.systemFont(ofSize: Constants.fontSize) rather than the existing location?

Reply:
Sure. Tested, and it looks the same, so I didn't update the screenshots.